### PR TITLE
internationalization "no results message" in MyData page

### LINKED ISSUE #8594: "No results were found" message on "My Data" page is not internationalized
**What steps does it take to reproduce the issue?**

* When does this issue occur?

When a user is on the "My Data" page and performs a search with no results, the "no results found" message is not translated. PR forthcoming from @JayanthyChengan. 

* Which page(s) does it occurs on?

My Data page (dataverseuser.xhtml?selectTab=dataRelatedToMe)

* To whom does it occur (all users, curators, superusers)?

All users with an account

**Which version of Dataverse are you using?**

5.8

**Screenshots:**

The My Data page with no search results found

### PR DESCRIPTION
**Pull request by JayanthyChengan**
**What this PR does / why we need it**:
When a search is performed on "My Data" page, the internationalization of message "no results found"  were fixed in this pull request.

**Which issue(s) this PR closes**:

Closes #8594

**Special notes for your reviewer**:

**Suggestions on how to test this**:

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

**Is there a release notes update needed for this change?**:

**Additional documentation**:

### CODE REVIEW

File: src/main/java/edu/harvard/iq/dataverse/mydata/DataRetrieverAPI.java [hunk 1]
```diff
@@ -384,7 +387,7 @@ public String retrieveMyDataAsJsonString(@QueryParam("dvobject_types") List<Stri
         List<String> filterQueries = this.myDataFinder.getSolrFilterQueries();
         if (filterQueries==null){
             logger.fine("No ids found for this search");
-            return this.getJSONErrorString(DataRetrieverAPI.MSG_NO_RESULTS_FOUND, null);
+            return this.getJSONErrorString(noMsgResultsFound, null);
```
Comment (sekmiller) [inline src/main/java/edu/harvard/iq/dataverse/mydata/DataRetrieverAPI.java:390]: Should we do something similar in MyDataFinder (lines 214 and 281) where it gets an error message from the bundle using DataRetrieverAPI.MSG_NO_RESULTS_FOUND?

Comment (sekmiller) [reply src/main/java/edu/harvard/iq/dataverse/mydata/DataRetrieverAPI.java:390]: Hey @JayanthyChengan, just wondering if you've had a chance to think about this. we'd like to get it merged along with your PR on large file internationalization. Thanks!

Comment (JayanthyChengan) [reply src/main/java/edu/harvard/iq/dataverse/mydata/DataRetrieverAPI.java:389]: Good Morning @sekmiller :  Please review the changes.  Thanks!

**Review by sekmiller — APPROVED** [review]
Looks good. Thanks for the contribution, @JayanthyChengan!

**Pull request merged**